### ISSUE TITLE
Import new data - Jilpanger 2017

### ISSUE THREAD

**Issue by pawilfahrt** [felt0134/drought-net-tickets#10]

**Comment by pawilfahrt** [comment]
Plan indicates 3 drought plots and 5 control plots. Check with Site PI

**Comment by pawilfahrt** [comment]
Biomass is divided into monocots/dicots/shrubs. Check cover data to see if there are non-graminoid monocots in data.

**Comment by pawilfahrt** [comment]
Kate is emailing site PI John Morgan

**Comment by KW529** [comment]
I got a response from Dr. Morgan on 1/4/2018: 
1)      In the site plan csv, there are 3 plots indicated as drought, but 5 listed as control. Is this correct? Or, were there 4 drought plots and 4 control plots? Yes, three as treatment and 5 as control - I wanted to have more controls given the few shelters we have erected. 

2)      In the biomass data, the biomass is sorted into ‘monocots’, ‘dicots’, and ‘woody’. This creates a slight issue because monocots includes all graminoids, but also some forbs like orchids, lilies, and some others. Did you all call all forbs dicots and all graminoids monocots?

We have sorted all monocots into one group - grasses and lillies and iridaceae as my volunteers can't tell the different most times. We don't cut orchids as our permit expressly prohibits us from doing so. Lillies and irids account for very little biomass at the site (maybe less than 5%) so the monocot biomass is mostly grass.
